Cryptopals/suraj/Set1Challenge1.c

```diff
@@ -0,0 +1,55 @@
+#include <stdio.h>
+#include <string.h>
+
+char input[] = "49276d206b696c6c696e6720796f757220627261696e206c696b65206120706f69736f6e6f7573206d757368726f6f6d";
+char output[200];
+char base64_lookup_table[64] = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789+/";
+
+
+unsigned int hex2dec(char x)
+{
+  char val = 0;

```

Comment:
Check what the function returns when x = 'a' . Go over each line in this function carefully.